tests/tests_tui/test_tui_selectdirectorytree_modalscreen.py

```diff
@@ -0,0 +1,75 @@
+import platform
+from pathlib import Path
+
+import pytest
+
+from datashuttle.tui.app import TuiApp
+from datashuttle.tui.screens.modal_dialogs import (
+    SelectDirectoryTreeScreen,
+)
+
+
+@pytest.mark.asyncio
+async def test_select_directory_tree(monkeypatch):
+    """
+    Test that changing the drive in SelectDirectoryTreeScreen
+    updates the DirectoryTree path as expected.
+    """
+
+    # Set up fake and real drives
+    actual_drive = (
```

Comment:
On further reflection, I think we can just have two separate names, `Drive 1`, `Drive 2` (as we are artificially setting the drive, it doesn't matter if they are real or not, as long as they are different